src/BinaryParsers/PEBinary/ProgramDatabase/LinkerCommandLine.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Globalization;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase
+{
+    /// <summary>Processes command lines stored by linkers in PDBs.</summary>
+    internal struct LinkerCommandLine
+    {
+        /// <summary>
+        /// The raw, unadulterated command line before processing.
+        /// </summary>
+        public readonly string Raw;
+
+        /// <summary>
+        /// Whether or not this command line enables incremental linking
+        /// </summary>
+        public readonly bool IncrementalLinking;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LinkerCommandLine"/> struct from a raw PDB-supplied command line.
+        /// </summary>
+        /// <param name="commandLine">The raw command line from the PDB.</param>
+        public LinkerCommandLine(string commandLine)
+        {
+            this.Raw = commandLine ?? "";
+            this.IncrementalLinking = false;
+
+            // https://docs.microsoft.com/cpp/build/reference/incremental-link-incrementally?view=msvc-170
+            bool debugSet = false;
+            bool optRef = false;
+            bool optIcf = false;
+            bool optLbr = false;
+            bool order = false;
+            bool explicitlyEnabled = false;
+            bool explicitlyDisabled = false;
+            foreach (string argument in ArgumentSplitter.CommandLineToArgvW(commandLine))
+            {
+                if (!CommandLineHelper.IsCommandLineOption(argument))
+                {
+                    continue;
+                }
+
+                // There are multiple /debug options so use StartsWith
+                if (argument.StartsWith("/debug", System.StringComparison.OrdinalIgnoreCase))
```

Comment:
>/ [](http://example.com/codeflow?start=41&length=1)

link.exe supports '-' as an arg prefix, is this symbol consistently normalized to '/' or do we need to handle the hyphen case here as well?

Reply:
Are we going to make a change here? i.e., support both '/debug' and '-debug'?

Reply:
Whoops missed this one.

Reply:
Fixed now.